src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsBaseHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import com.amazonaws.services.stepfunctions.AWSStepFunctions;
+import com.amazonaws.services.stepfunctions.AWSStepFunctionsClientBuilder;
+import com.github.piedpiper.aws.AccessKeySecretKeyCredentialProvider;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+
+public abstract class StepFunctionsBaseHandler extends StepFunctionsBaseNode {
+	protected AWSStepFunctions stepFunctionsClient; 
+	private void setStepFunctionsClient(NodeInput input) throws Exception{
```

Comment:
bad use of a setter. Setters should also set a parameter and the parameted type and settign variable type shoul dbe the same according to java bean conventions.